### PR TITLE
Berdikulov/dtos

### PR DESCRIPTION
**Pull request by Berdikulov-571**

### CODE REVIEW

**Review by murodovich — APPROVED** [review]

File: src/Application/DataTransferObjects/Courses/CourseCreationDTO.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿namespace Application.DataTransferObjects.Courses
+{
+    public record CourseCreationDTO
+    (
+        string Name,
+        long SpecialityId,
+        TimeOnly LessonStartAt,
+        TimeOnly LessonEndAt,
+        DateOnly StartAt,
+        DateOnly EndAt
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Courses/CourseCreationDTO.cs]: List<long> Teachers

Shuni qo'shvorelik

File: src/Application/DataTransferObjects/Courses/CourseModificationDTO.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Application.DataTransferObjects.Courses
+{
+    public record CourseModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Courses/CourseModificationDTO.cs]: Id shartmas Id routedan keladi

File: src/Application/DataTransferObjects/Courses/SpecialtyModificationDTO.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Application.DataTransferObjects.Courses
+{
+    public recor SpecialtyModificationDTO 
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Courses/SpecialtyModificationDTO.cs]: Id shartmas

File: src/Application/DataTransferObjects/HomeWorks/HomeworkFileModificationDTO.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Application.DataTransferObjects.HomeWorks
+{
+    public record HomeworkFileModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/HomeWorks/HomeworkFileModificationDTO.cs]: Id shartmas

File: src/Application/DataTransferObjects/HomeWorks/HomeworkModificationDTO.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Application.DataTransferObjects.HomeWorks
+{
+    public record HomeworkModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/HomeWorks/HomeworkModificationDTO.cs]: Id shartmas

File: src/Application/DataTransferObjects/Lessons/AttandanceModificationDTO.cs [hunk 6]
```diff
@@ -0,0 +1,13 @@
+﻿using Domain.Enums;
+
+namespace Application.DataTransferObjects.Lessons
+{
+    public record AttandanceModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Lessons/AttandanceModificationDTO.cs]: Id shart emas

File: src/Application/DataTransferObjects/Lessons/LessonModificationDTO.cs [hunk 7]
```diff
@@ -0,0 +1,10 @@
+﻿namespace Application.DataTransferObjects.Lessons
+{
+    public record LessonModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Lessons/LessonModificationDTO.cs]: Id shartmas

File: src/Application/DataTransferObjects/Lessons/VideoCreationDTO.cs [hunk 8]
```diff
@@ -0,0 +1,13 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Application.DataTransferObjects.Lessons
+{
+    public record VideoCreationDTO
+    (
+        IFormFile Path,
+        string Size,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Lessons/VideoCreationDTO.cs]: File size kerakmas

File: src/Application/DataTransferObjects/Lessons/VideoModificationDTO.cs [hunk 9]
```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Application.DataTransferObjects.Lessons
+{
+    public record VideoModificationDTO
+    (
+        long Id,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Lessons/VideoModificationDTO.cs]: Id shartmas

File: src/Application/DataTransferObjects/Lessons/VideoModificationDTO.cs [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+        IFormFile? Path,
+        string Size,
```
Comment (SardorSohinazarov) [inline src/Application/DataTransferObjects/Lessons/VideoModificationDTO.cs]: Size shartmas

**Review by SardorSohinazarov — APPROVED** [review]

**Pull request merged**